packages/monitor-v2/src/bot-oo-v3/index.ts

```diff
@@ -0,0 +1,50 @@
+import { delay } from "@uma/financial-templates-lib";
+import { BotModes, initMonitoringParams, Logger, startupLogLevel } from "./common";
+import { settleAssertions } from "./SettleAssertions";
+
+const logger = Logger;
+
+async function main() {
+  const params = await initMonitoringParams(process.env);
```

Comment:
the implications of this implementation are that we need to deploy one bot per chain the OO is on. this means having, at minimum: a) Mainnet, b) optimism c) polygon d) arbitrum.

we could also design this bot to take in a set of chain IDs and then iterate over all of them so one bot handels all chains.

Note I dont think we should actually do this...just pointing it out that this tradeoff, while making the code easier to read makes the configs longer.

Reply:
thinking more on this lets leave it as you have it here. its easier to manage and is more consistant with other setups.